recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/alsa-state/files/asound.state

```diff
@@ -0,0 +1,321 @@
+state.rsnddai0ak4613h {
```

Comment:
Where contents of this file comes from?

Reply:
File was generated by 'alsactl store' on XS board after setting volume by alsamixer.

Reply:
Should this be mentioned in commit description?

Reply:
It seems that this patch will affect all supported machines which are supported in a prod-devel at the moment. Are you sure that this config is appropriate for all modifications (Salvator-X/Salvator-XS boards with H3/M3 installed)?

Reply:
Good point. We see it works on X/H3, XS/H3. Will check other devices.

Reply:
I think you need to provide minimal "asound.state", which achieves your needs. 

And yes, it would be cool to have more elaborate commit description.

Reply:
Addressing comments.
1) "minimal" - I have checked behavior with asound.state containing only required "device.7". Settings loaded as expected, but after shutdown of DomD, file is rewritten by service to same state as it is provided by this patch. Also it's not specified that alsactl will properly handle not complete config file. So, my proposal is to take complete asound.state as provided in patch.
2) Test on -X/-XS boards with H3/M3 installed. Config tested on -X/H3 and -XS/H3 boards. My understanding is that we have same audio HW on both boards, and CPU will not affect audio settings. So proposal is to take patch without test with M3 CPU.

Reply:
Forgot to mention:
* Commit message improved, and now contains explanation of how file was generated.
* All devices except device.7 has initial (default) settings, provided by alsactl. Device.7 has value required for proper usage of sound output.